src/Lexer.test.ts

```diff
@@ -0,0 +1,59 @@
+// @author Anush kumar N <anush.kumar@codingmart.com>
+
+import Lexer, { literalWithType } from './Lexer';
+import TokenType from './Token';
+
+describe('Lexer', () => {
```

Comment:
What about a test case with an empty string input?

Reply:
if the input is empty, then the lexer would return `{ type: illegal, literal: '' }`

Reply:
oops! It will return `eof` on the first `l.nextToken()`

I forgot to update the comment.

Reply:
oops! It will return `eof` on the first `l.nextToken()`

I forgot to update the comment.